docs/src/cisos/security/communication-matrix.md

```diff
@@ -0,0 +1,96 @@
+
+| No.  |                 Component                 | Source Zone  | Protocol |  Destination   |  Destination Zone   | Port |  C  |  I  | Auth | Trust |   Purpose    |                    Notes                    |
+| :--: | :---------------------------------------: | :----------: | :------: | :------------: | :-----------------: | :--: | :-: | :-: | :--: | :---: | :----------: | :-----------------------------------------: |
+| G1.1 |                 metal-ccm                 | Seed Cluster |  HTTPS   |   metal-api    | Metal Control Plane | 443  |  x  |  x  |  x   |   x   | API Requests |       Used for management operations.       |
+| G2.1 |        firewall-controller-manager        | Seed Cluster |  HTTPS   |   metal-api    | Metal Control Plane | 443  |  x  |  x  |  x   |   x   | API Requests |        Used for firewall management.        |
+| G2.2 |        firewall-controller-manager        | Seed Cluster |  HTTPS   | kube-apiserver |    Seed Cluster     | 443  |  x  |  x  |  x   |   x   | API Requests |        Used for firewall management.        |
+| G2.3 |        firewall-controller-manager        | Seed Cluster |  HTTPS   | kube-apiserver |    Shoot Cluster    | 443  |  x  |  x  |  x   |   x   | API Requests |        Used for firewall management.        |
+| G3.1 |            firewall-controller            |   Firewall   |  HTTPS   | kube-apiserver |    Seed Cluster     | 443  |  x  |  x  |  x   |   x   | API Requests |        Used for firewall management.        |
+| G3.2 |            firewall-controller            |   Firewall   |  HTTPS   | kube-apiserver |    Shoot Cluster    | 443  |  x  |  x  |  x   |   x   | API Requests |        Used for firewall management.        |
+| G3.3 |            firewall-controller            |   Firewall   |  HTTPS   | Controller URL |      Internet       | 443  |  x  |  x  |      |       | Self-Update  | Controller URL and version provided by FCM. |
+| G4.1 | machine-controller-manager-provider-metal | Seed Cluster |  HTTPS   |   metal-api    | Metal Control Plane | 443  |  x  |  x  |  x   |       | API Requests |       Used for management operations.       |
+| G5.1 |     gardener-extension-provider-metal     | Seed Cluster |  HTTPS   |   metal-api    | Metal Control Plane | 443  |  x  |  x  |  x   |       | API Requests |       Used for management operations.       |
+| G5.2 |     gardener-extension-provider-metal     | Seed Cluster |  HTTPS   | kube-apiserver |   Garden Cluster    | 443  |  x  |  x  |  x   |       | API Requests |       Used for management operations.       |
+| G5.3 |     gardener-extension-provider-metal     | Seed Cluster |  HTTPS   | kube-apiserver |    Seed Cluster     | 443  |  x  |  x  |  x   |       | API Requests |       Used for management operations.       |
+| G5.4 |     gardener-extension-provider-metal     | Seed Cluster |  HTTPS   | kube-apiserver |    Shoot Cluster    | 443  |  x  |  x  |  x   |       | API Requests |       Used for management operations.       |
+
```

Comment:
Theoretically there are more extensions that we have, but maybe it's enough already. :D